webapp/config.py

```diff
@@ -65,6 +66,12 @@ def ALMA_SAP_INVOICES_ECS_TASK_DEFINITION(self) -> str:
     def ALMA_SAP_INVOICES_CLOUDWATCH_LOG_GROUP(self) -> str:
         return self._getenv("ALMA_SAP_INVOICES_CLOUDWATCH_LOG_GROUP")
 
+    @property
+    def LOGIN_DISABLED(self) -> bool:
+        if self._getenv("LOGIN_DISABLED").lower() == "true":  # noqa: SIM103
+            return True
+        return False
```

Comment:
+1 to defaulting to `False` if not explicitly overridden, thereby making login _enabled_ (required).

Reply:
Yes, I think `LOGIN_DISABLED` should be `False` unless explicitly set as `True`. Infra will set up Terraform to pull "true" or "false" from an SSM parameter. ✨